cuda_core/pyproject.toml

```diff
@@ -46,8 +46,8 @@ dependencies = [
 ]
 
 
-[tool.setuptools]
-packages = ["cuda", "cuda.core"]
+[tool.setuptools.packages.find]
+include = ["cuda.core*"]
```

Comment:
Note: This change (and those in `setup.py`) not only makes it cleaner but is also necessary when adding nested modules to a namespace package. It is a quite subtle `setuptools` nonsense, as we are forced to do this:
> Note: specifying `ext_modules` might also prevent auto-discover from taking place, unless your opt into [Configuring setuptools using pyproject.toml files](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html) (which will disable the backward compatible behaviour).

https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#automatic-discovery

The existing treatment added when `cuda.core` was checked in is simply not working...